code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -549,7 +549,9 @@ GLOBAL_VAR(medibot_unique_id_gen)
 						healies *= 1.5
 					if(treatment_method == TOX && HAS_TRAIT(patient, TRAIT_TOXINLOVER))
 						healies *= -1.5
-					if(emagged == 2)
+					if(alt_heal(patient))
```

Comment:
why not just make an overridable heal proc and move all the code out there, so it can be fully overriden

Reply:
wdym? sory